modules/footer/Hackathon2023Footer.tsx

```diff
@@ -0,0 +1,11 @@
+import { ReactElement } from 'react';
```

Comment:
I wasn't sure if I should put this in the `footer` folder or in the `hackathon-2023` folder but I think it makes sense to put all the footers together haha. 

Also, I'm not sold on the prefix of `Hackathon2023` being before each of them so open to suggestions here on variable naming haha.

Reply:
I think we can keep the `Hackathon2023Footer` in `hackathon-2023`. The component itself could render the main `Footer` component with a property that toggles the new styles on. I could see us maybe wanting a 3 column footer layout for other pages in the future.

Reply:
I think that sounds good! I'll add that as a note for whoever tackles the Footer task to do that then!